playbooks/roles/logstash/tasks/main.yml

```diff
@@ -72,3 +82,14 @@
     minute={{ logstash_optimize_cron.minute }}
     job="/usr/bin/python {{ logstash_app_dir }}/share/logstash-elasticsearch-scripts/logstash_index_optimize.py -d {{ LOGSTASH_DAYS_TO_KEEP }} > {{ logstash_log_dir }}/optimize_cron"
   when: LOGSTASH_ROTATE|bool
+
+- name: plugins | installing logstash plugins
+  command: "sudo /opt/logstash/bin/logstash-plugin install {{ item }}"
+  notify: "restart logstash"
+  with_items:
+    logstash-input-beats
```

Comment:
We should pull these from a variable, like

```yaml
# playbooks/roles/logstash/defaults/main.yml
LOGSTASH_PLUGINS:
  - logstash-input-beats
  - logstash-filter-multiline
  - logstash-output-file

# playbooks/roles/logstash/tasks/main.yml
# ...
  with_items: LOGSTASH_PLUGINS
```

This makes it easy to customize the plugins without modifying the role.